src/amo/reducers/reviews.js

```diff
@@ -576,6 +579,12 @@ export default function reviewsReducer(
       }
       return newState;
     }
+    case SET_VIEW_CONTEXT: {
```

Comment:
I see what you were trying to do here but relying on this action is fragile. It's not guaranteed that `setViewContext()` will be dispatched and, actually, I think it's a bug that it is (the context is the same between both views). 

`LOCATION_CHANGE` would be better because we know it gets dispatched whenever the URL changes. Example:

https://github.com/mozilla/addons-frontend/blob/40d1ba4e8fe83bfdcf4bc44ba3854a5ba8a4ea61/src/core/reducers/errorPage.js#L59